portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -69,10 +76,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
       // Get the input from the form
-      String comment = request.getParameter("comment");
-      Entity commentEntity = new Entity("Comment");
-      commentEntity.setProperty("comment", comment);
-      datastore.put(commentEntity);
+      if (!request.getParameter("comment").isEmpty()) {
```

Comment:
Should handle the case where `getParameter()` returns `null`, probably treating it similar to empty.